nasa-veda-platform/adding-content/docs-and-notebooks.qmd

```diff
@@ -28,6 +28,14 @@ There are two template notebooks in this section that you can use as a starting
 - Only include imports that are needed for the notebook to run.
 - We don't enforce any formatting, but periodically run black on all the notebooks. If you would like to run black yourself do `pip install black[jupyter]` and then `black`.
```

Comment:
Is this still true given we want people to use the pre-commit?

Reply:
I think we can keep we periodically run black on all the notebooks, since that is included in pre-commit. I'll update the second sentence to link to the pre-commit instructions.